lumicks/pylake/detail/image.py

```diff
@@ -114,8 +114,11 @@ def line_timestamps_image(time_stamps, infowave, pixels_per_line):
     pixels_per_line : int
         The number of pixels on the fast axis of the scan.
     """
-    min_stamps = reconstruct_image(time_stamps, infowave, shape=(pixels_per_line,), reduce=np.min)
-    return min_stamps.astype(np.int64)[:, 0]
```

Comment:
the conversion error happens here?

Reply:
Conversion happens in two places in `reconstruct_image`. Line `161` and `217` in `image.py`.

See the branch `dtype_experimental`.